src/battle_script_commands.c

```diff
@@ -6434,6 +6486,18 @@ static void Cmd_moveend(void)
             if (B_CHARGE <= GEN_8 || moveType == TYPE_ELECTRIC)
                 gStatuses3[gBattlerAttacker] &= ~(STATUS3_CHARGED_UP);
             memset(gQueuedStatBoosts, 0, sizeof(gQueuedStatBoosts));
+
+            for (i = 0; i < gBattlersCount; i++)
+            {
+                if (gBattleStruct->commanderActive[i] != SPECIES_NONE && !IsBattlerAlive(i))
+                {
+                    u32 partner = BATTLE_PARTNER(i);
+                    gBattleStruct->commanderActive[i] = SPECIES_NONE;
+                    if (IsBattlerAlive(partner))
+                        gStatuses3[partner] &= ~STATUS3_COMMANDER;
+                }
+            }
+
```

Comment:
Move this to `MOVEEND_MULTIHIT_MOVE` to fix "Tatsugiri gets hit by Dragon Darts when a commanded Dondozo faints". Thankfully `MOVEEND_MULTIHIT_MOVE` is after `MOVEEND_NEXT_TARGET` so it shouldn't fail the other tests

Reply:
see above.